openlibrary/components/AuthorIdentifiers.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div>
+        <div class="label"><label for="identifiers-display">Identifiers</label></div>
+        <div class="allButHeader">
+            <div id="identifiers-display">
+                <div class="wrapper">
+                    <span class="box">
+                        <select v-model="selected" name="name" aria-invalid="false">
+                            <option disabled value="">Select one</option>
+                            <option v-for="item in allIdentifiers" :key="item.name" :value="item.name">{{item.label}}</option>
+                        </select>
+                    </span>
+                    <span class="box">
+                        <input type="text" name="value" id="id-value" v-model="inputValue" aria-invalid="false" @keyup.enter=setIdentifier>
+                    </span>
+                    <span class="box">
+                        <button type="button" name="set" :disabled="!setButtonEnabled" @click=setIdentifier>Set</button>
+                    </span>
+                    <template v-for="(item) in identifiersWithValues">
+                        <div class="box" :key="item.name">{{ item.label }}</div>
+                        <div class="box" :key="item.name">{{ item.value }}</div>
+                        <div class="box" :key="item.name">
+                          <button type="button" @click="removeIdentifier(item.name)">Remove</button>
+                        </div>
+                    </template>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    // Props are for external options; if a subelement of this is modified,
+    // the view automatically re-renders
+    props: {
+        // this is the list of remote ids currently associated with the book in the database
+        // It is passed as a string and then converted to json
+        remote_ids_string: {
```

Comment:
I agree it should be renamed, however the render component function for some reason makes everything lowercase. I poked around in the python a bit and really don't understand why it would be made lowercase. Do you have any ideas?

Reply:
Oh! Interesting; not sure why that is. Not worth getting hung up on right now.